README.md

```diff
@@ -929,7 +937,8 @@ IPO? What is a lockup period?
 
 ### Sales and liquidity
 
-❗️ With private companies, it can be very hard to know the value of equity.
+❗️ With private companies, it can be very hard to know the
+[value of equity](https://www.investopedia.com/articles/fundamental-analysis/11/valuing-private-companies.asp).
```

Comment:
I'm moving this to the section we have on that, and forward linking to that section.